core/pom.xml

```diff
@@ -616,6 +621,48 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-shade-plugin</artifactId>
```

Comment:
@gengliangwang  It actually conflicts with parent pom.xml which causes the jetty can not be shaded into spark-core. The current master branch can not start Spark UI, see error message:

```
Setting default log level to "WARN".
To adjust logging level use sc.setLogLevel(newLevel). For SparkR, use setLogLevel(newLevel).
22/12/09 14:58:54 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
java.lang.NoClassDefFoundError: org/eclipse/jetty/util/thread/Scheduler
  at org.apache.spark.ui.SparkUI.<init>(SparkUI.scala:67)
  at org.apache.spark.ui.SparkUI$.create(SparkUI.scala:223)
  at org.apache.spark.SparkContext.<init>(SparkContext.scala:484)
  at org.apache.spark.SparkContext$.getOrCreate(SparkContext.scala:2739)
  at org.apache.spark.sql.SparkSession$Builder.$anonfun$getOrCreate$2(SparkSession.scala:978)
  at scala.Option.getOrElse(Option.scala:189)
  at org.apache.spark.sql.SparkSession$Builder.getOrCreate(SparkSession.scala:972)
  at org.apache.spark.repl.Main$.createSparkSession(Main.scala:106)
  ... 55 elided
Caused by: java.lang.ClassNotFoundException: org.eclipse.jetty.util.thread.Scheduler
  at java.net.URLClassLoader.findClass(URLClassLoader.java:387)
  at java.lang.ClassLoader.loadClass(ClassLoader.java:419)
  at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:352)
  at java.lang.ClassLoader.loadClass(ClassLoader.java:352)
  ... 63 more
<console>:14: error: not found: value spark
       import spark.implicits._
              ^
```

Reply:
maybe should add `combine.children="append"` to `<relocations>` tag

Reply:
Ya, I also saw this when I build distributions. Thank you, @pan3793 !